src/ui/chat-window.js

```diff
@@ -18,11 +18,75 @@ export class ChatScene extends Phaser.Scene {
 
     col1 = 0;
     visible = false;
+    isPromptOpen = false;
 
     constructor() {
         super({ key: CONSTANTS.SCENES.CHAT });
     }
 
+    // TODO: if we need more prompts we should reuse a lot of this
+    // Right now it is specifically for selling X amount of an item
+    prompt(promptText, item) {
+        let originallyVisible = this.visible;
```

Comment:
Small thing, we should use const unless we plan on modifying the value. Not specifically for performance reasons but for ease of understanding the code

Reply:
Updated! https://github.com/EthanVieira/rs-clicker/pull/56/commits/ca5f638ee5466c01f3ba32b12dcd04aaceda9e03